docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -4,17 +4,20 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 
-[id="getting-started-dev-services-tutorial"]
```

Comment:
You can keep that, what you need is to get rid of the new line above.

See https://github.com/quarkusio/quarkusio.github.io/commit/a30a2d57989e994f2c90b57468ed52fad8f87116 and https://quarkus.io/guides/getting-started-dev-services